diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -292,6 +296,32 @@ namespace diff_drive_controller{
       cmd_vel_pub_.reset(new realtime_tools::RealtimePublisher<geometry_msgs::TwistStamped>(controller_nh, "cmd_vel_out", 100));
     }
 
+    // Wheel data:
+    if (publish_wheel_data_)
+    {
+      wheel_data_pub_.reset(new realtime_tools::RealtimePublisher<WheelDataStamped>(controller_nh, "wheel_data", 100));
+
+      wheel_data_pub_->msg_.header.frame_id = "0";
```

Comment:
What is the point of setting this?

Reply:
none, that's a miss from melding... About to remove

Reply:
Lack of tests ?
You are hurting [diff_drive_publish_wheel_data_test.cpp](https://github.com/ros-controls/ros_controllers/pull/316/files#diff-1a8513c9c34436991145f42a03d65da8) feelings :D 
You want moar ?